src/main/resources/strings.tsv

```diff
@@ -270,4 +270,189 @@ isoCountries	WF	Wallis and Futuna Islands
 isoCountries	EH	Western Sahara
 isoCountries	YE	Yemen
 isoCountries	ZM	Zambia
-isoCountries	ZW	Zimbabwe
\ No newline at end of file
+isoCountries	ZW	Zimbabwe
+isoLanguages	ab	Abkhaz
```

Comment:
@bradsorour please clarify with Jag for the approach on having isoLanguage code -> label translation. Maybe we can keep it in one place in CCD

Reply:
I think this is a good idea to avoid all this code duplication. I am, however, running out of time to implement this change. Jag would possibly need to implement this (together with yourself) before deployment ad part of submit hearing requirements.